int32 customer_nationality
int32 favourite_taste
int32 customer_number